app.js

```diff
@@ -1,7 +1,30 @@
-const signInForm = document.getElementById('sign-in');
+import { signUpUser, signInUser, getUser, redirectIfLoggedIn, checkAuth } from '/fetch-utils.js';
 
+const signInForm = document.getElementById('sign-in');
 const signUpForm = document.getElementById('sign-up');
 
```

Comment:
somewhere in this file you should use the `getUser()` and `redirectIfLoggedIn()` function  outside of a form to determine whether or not a user is logged in. If the user is logged in, you should redirect them to the homepage. An authenticated user should not be able to access this page to sign in.